curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8e41e2f190c58404dd46e.md

```diff
@@ -6,8 +6,7 @@ dashedName: step-118
 ---
 
 # --description--
-
-On a new line, add the string `" You attack it with your <weapon>."` to the `text` value, replacing `<weapon>` with the player's current weapon. Additionally, remember that this line of text starts with a space so it will properly display. 
+On a new line, append the string `" You attack it with your <CURRENT_WEAPON_NAME>."` to the current value of `text`, and replace `<CURRENT_WEAPON_NAME>` with the player's current weapon. Additionally, remember that this new string starts with a space to display it properly.
```

Comment:
Let's go with these changes mentioned in the issue here

https://github.com/freeCodeCamp/freeCodeCamp/issues/53601#issuecomment-1933252876

Reply:
I did it, please check 
Also, I will wait for someone's review to new issues before making any change.

Thank you very much